README.md

```diff
@@ -0,0 +1,15 @@
+# Phonebook
+Mozilla's internal phonebook app.
+
+## Installation
+* Clone, host on a local, PHP-enabled server, run.
+* Optional: copy ``config-local.php-dist`` to ``config-local.php`` and tweak as needed
+
+*Note:* The app requires LDAP server access. You probably need a VPN connection up and running.
```

Comment:
Link to VPN mana/whatever page detailing how to connect would be good here; I actually forget which VPN we're even supposed to use for this anymore (IIRC office VPNs were decommissioned, but then spun back up for folks who needed them for things--which one do I use?).